homeassistant/components/tuya/select.py

```diff
@@ -242,6 +242,12 @@
             icon="mdi:timer-cog-outline",
             translation_key="countdown",
         ),
+        SelectEntityDescription(
+            key=DPCode.MIST,
+            name="Mist",
```

Comment:
Needs to use translations

Reply:
As my setup is English (only), I cannot test translations - is adding a translation-key sufficient?

Reply:
Adding support for translations is mandatory for any new addition to our codebase..

Reply:
support for translations was added